docker/build/installers/install_pcl.sh

```diff
@@ -31,19 +35,34 @@ if ldconfig -p | grep -q libpcl_common ; then
     exit 0
 fi
 
+GPU_OPTIONS="-DCUDA_ARCH_BIN=\"${SUPPORTED_NVIDIA_SMS}\""
+if [ "${WORKHORSE}" = "cpu" ]; then
+    GPU_OPTIONS="-DWITH_CUDA=OFF"
+fi
+
+info "GPU Options for PCL:\"${GPU_OPTIONS}\""
+
+TARGET_ARCH="$(uname -m)"
+ARCH_OPTIONS=""
+if [ "${TARGET_ARCH}" = "x86_64" ]; then
+    ARCH_OPTIONS="-DPCL_ENABLE_SSE=ON"
```

Comment:
Does this cause compatibility problems between different machines?

Reply:
> between

This setting was based on https://src.fedoraproject.org/rpms/pcl/blob/master/f/pcl.spec.
I think the main reason for doing so was, SSE settings were meaningless for non-x86 CPU architectures.

Reply:
But I remember that dreamview module might generate coredump on some machines if SSE is enabled.

Reply:
sorry, I didn't notice that you fixed it by the patch